tests/factories.py

```diff
@@ -33,16 +33,16 @@ class Meta:
 
 
 class ActionReportFactory(factory.Factory):
-    """Generate Action reports."""
+    """Generate Action reports."""  # noqa: DCO060
```

Comment:
Out of curiosity, what are these "noqa: DCO060"?

Reply:
@arturo-seijas This is to let the docstrings without pass the linting stage `flake8-docstrings-complete` pass.
Will add a comment to avoid confusion! Thanks for the feedback!